### PR TITLE
python310Packages.airtouch4pyapi: 1.0.5 -> 1.0.8

### PR DESCRIPTION
**Pull request by dotlambda**
###### Description of changes
Diff: https://github.com/LonePurpleWolf/airtouch4pyapi/compare/3478388...v1.0.8

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/airtouch4pyapi/default.nix [hunk 1]
```diff
@@ -8,27 +8,18 @@
...
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "LonePurpleWolf";
     repo = pname;
-    # https://github.com/LonePurpleWolf/airtouch4pyapi/issues/5
-    rev = "34783888846783c058fe79cec16feda45504f701";
-    sha256 = "17c7fm72p085pg9msvsfdggbskvm12a6jlb5bw1cndrqsqcrxywx";
+    rev = "v${version}";
```
Comment (fabaff) [inline pkgs/development/python-modules/airtouch4pyapi/default.nix:19]:
```suggestion
    rev = "refs/tags/v${version}";
```

File: pkgs/development/python-modules/airtouch4pyapi/default.nix [hunk 2]
```diff
@@ -8,27 +8,18 @@
...
+    hash = "sha256-RiRwebumidn0nijL/e9J74ZYx0DASi1up5BTNxYoGEA=";
   };
 
-  patches = [
-    # https://github.com/LonePurpleWolf/airtouch4pyapi/pull/10
-    (fetchpatch {
```
Comment (fabaff) [inline pkgs/development/python-modules/airtouch4pyapi/default.nix:26]: `fetchpatch` can be removed from the inputs.

**Comment by wegank** [comment]
@ofborg build home-assistant-component-tests.airtouch4

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 213417` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.airtouch4</li>
    <li>python310Packages.airtouch4pyapi</li>
    <li>python311Packages.airtouch4pyapi</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
Change looks good to me.

**Pull request merged**